src/part05.rs

```diff
@@ -47,7 +47,8 @@ impl BigInt {
         }
     }
 
-    // We can convert any vector of digits into a number, by removing trailing zeros. The `mut`
+    // Any vector of digits, which meets the structure of BigInt's `data` field, can be easily
```

Comment:
This makes it sound like conversion is possible only with those specific vectors that do meet the structure, but the entire point here is that we do not have to assume anything about the vector.

Could you elaborate on what you found confusing here? Maybe that helps us to find a better wording.